sources/api/migration/migrations/v1.1.3/cpu-manager-policy/src/main.rs

```diff
@@ -0,0 +1,84 @@
+#![deny(rust_2018_idioms)]
+
+use migration_helpers::{error, migrate, Migration, MigrationData, Result};
+use snafu::ResultExt;
+use std::fs;
+use std::io;
+use std::path::Path;
+use std::process;
+
+const SETTING: &str = "settings.kubernetes.cpu-manager-policy";
+const CPU_MANAGER_POLICY_CHECKPOINT: &str = "var/lib/kubelet/cpu_manager_state";
+/// We remove cpu manager policy checkpoint value if checkpoint value is differenet from upgrade version
+/// or downgrade version cpu-manager-policy setting value.
+pub struct CpuManagerPolicyCleaner;
+
+impl Migration for CpuManagerPolicyCleaner {
+    fn forward(&mut self, input: MigrationData) -> Result<MigrationData> {
+        // removing existing cpu_manager_policy_state file
+        if Path::new(CPU_MANAGER_POLICY_CHECKPOINT).exists() {
+            println!(
+                "Deleting existing cpu manager policy checkpoint: '{}'",
+                CPU_MANAGER_POLICY_CHECKPOINT
+            );
+            match fs::remove_file(CPU_MANAGER_POLICY_CHECKPOINT) {
+                Ok(()) => {}
+                Err(e) => {
+                    if e.kind() != io::ErrorKind::NotFound {
+                        return Err(e).context(error::RemoveFile {
+                            path: CPU_MANAGER_POLICY_CHECKPOINT,
+                        });
+                    } else {
+                        println! ("NotFound: '{}'", CPU_MANAGER_POLICY_CHECKPOINT)
+                    }
+                }
+            }
+        }
+        Ok(input)
+    }
+
+    fn backward(&mut self, mut input: MigrationData) -> Result<MigrationData> {
+        // removing cpu-manager-policy setting, which old version doesn't know about it.
+        if let Some(data) = input.data.remove(SETTING) {
+            println!("Removed {}, which was set to '{}'", SETTING, data);
+        } else {
+            println!("Found no {} to remove", SETTING);
+        }
```

Comment:
I'd prefer to handle removing the setting together with `cpu-manager-reconcile-period`, and leave this migration purely for the unusual checkpoint fix.

Maybe name them:
* kubelet-cpu-manager
* kubelet-cpu-manager-state